.github/workflows/test.yml

```diff
@@ -52,7 +52,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: ./.github/setup-docker-compose
+      - id: setup
+        uses: ./.github/setup-docker-compose
+      - name: Copy index.html for Django templates
+        run: |
+          mkdir -p dist
+          docker cp $(docker create ${{ steps.setup.outputs.tag }}):/app/dist/prod/index.html ./dist/
```

Comment:
It only started being a problem when I modified [this test](https://github.com/oddbird/MetaDeploy/blob/00ec8585f248e4d42ead830500274d0d77f3624d/metadeploy/tests/context_processors.py#L7) to use the client instead of the request factory. I think that's the only non-API view we're calling during testing and it was complaining about `index.html` not existing.

IMO testing with the client is more robust and realistic than request factory because we're also testing that middleware and context processors are applied correctly

Reply:
Oh, and Metecho doesn't need it because it still uses `rf` for that test https://github.com/oddbird/Metecho/blob/main/metecho/tests/context_processors.py#L9